common-template/src/test/java/drawer/TestErrataNotificationsTemplate.java

```diff
@@ -0,0 +1,33 @@
+package drawer;
+
+import com.redhat.cloud.notifications.ingress.Action;
+import com.redhat.cloud.notifications.qute.templates.IntegrationType;
+import com.redhat.cloud.notifications.qute.templates.TemplateDefinition;
+import com.redhat.cloud.notifications.qute.templates.TemplateService;
+import helpers.ErrataTestHelpers;
+import io.quarkus.test.junit.QuarkusTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@QuarkusTest
+class TestErrataNotificationsTemplate {
+
+    static final String BUGFIX_ERRATA = "new-subscription-bugfix-errata";
+    private static final Action ACTION = ErrataTestHelpers.createErrataAction();
+
+    @Inject
+    TemplateService templateService;
+
+    @Test
```

Comment:
For those tests, you can use `@ParameterizedTest` annotation to test each errata templates for each integration type like what we have [here](https://github.com/RedHatInsights/notifications-backend/blob/master/common-template/src/test/java/drawer/TestRbacTemplate.java).
Because rendered templates for those three event types are almost identical, the test should be quite easy to understand/maintain.